docs/modules/ROOT/examples/operator/bundle.yaml

```
@@ -46,9 +46,6 @@ spec:
                 properties:
                   discoveryServiceType:
                     description: Type of the service used to discover Hazelcast cluster.
-                    enum:
-                    - LoadBalancer
-                    - NodePort
                     type: string
                   memberAccess:
                     description: 'Method of how each member is accessed from the external client. Valid values are: - "NodePortExternalIP" (default): each member is accessed by the NodePort service and the node external IP/hostname - "NodePortNodeName": each member is accessed by the NodePort service and the node name - "LoadBalancer": each member is accessed by the LoadBalancer service external address'
```

```suggestion
                    description: 'How each member is accessed from the external client. Valid values are: - "NodePortExternalIP" (default): each member is accessed by the NodePort service and the node external IP/hostname - "NodePortNodeName": each member is accessed by the NodePort service and the node name - "LoadBalancer": each member is accessed by the LoadBalancer service external address'
```

Could we define the valid values like this for `externalConnectivity` field for Management Center?

---

In `externalConnectivity` field the valid values are resources that are created by Kubernetes, so those values are known by everyone. However terms `NodePortExternalIP`, `NodePortNodeName` are made up by us, so it was needed to explain these in detail.